api/file/src/mediaelement.js.html

```diff
@@ -383,7 +404,7 @@
      * @param {number} length How many subranges to break the waveform into.
      * @param {number} first First sample in the required range.
      * @param {number} last Last sample in the required range.
-     * @return {number[]|number[][]} Array of 2*&lt;length&gt; peaks or array of
+     * @return {number[]|Number.&lt;Array[]&gt;} Array of 2*&lt;length&gt; peaks or array of
```

Comment:
If I'm reading this correctly, the type is `Number.<Array[]>`. I'm not familiar with that typing syntax, is it correct?

Reply:
yes that's their doc syntax for a multi dimensional number array. annoying but cest la vie